packages/paypal-commerce-integration/src/paypal-commerce-alternative-methods/paypal-commerce-alternative-method-ratepay-payment-strategy.ts

```diff
@@ -0,0 +1,233 @@
+import {
+    InvalidArgumentError,
+    OrderFinalizationNotRequiredError,
+    OrderRequestBody,
+    PaymentArgumentInvalidError,
+    PaymentInitializeOptions,
+    PaymentIntegrationService,
+    PaymentMethodInvalidError,
+    PaymentRequestOptions,
+    PaymentStrategy,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+import { LoadingIndicator } from '@bigcommerce/checkout-sdk/ui';
+
+import PayPalCommerceIntegrationService from '../paypal-commerce-integration-service';
+
+import { WithPayPalCommerceAlternativeMethodsPaymentInitializeOptions } from '../index';
+import { BirthDate, PayPalCommerceInitializationData } from '../paypal-commerce-types';
+import { PaypalCommerceRatePay } from './paypal-commerce-alternative-methods-payment-initialize-options';
+
+export default class PayPalCommerceAlternativeMethodRatePayPaymentStrategy
```

Comment:
It might be better to rename the strategy from `PayPalCommerceAlternativeMethodRatePayPaymentStrategy` to `PayPalCommerceRatepayPaymentStrategy`. So in this case you will have ratepay own separated strategy, ints own initialization options and tests

Reply:
It might be easier to understand whats going on in the future

Reply:
Renamed